index.html

```diff
@@ -0,0 +1,42 @@
+           <div id="capture">
+               <caption id="img-caption">Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit</caption>
+           </div>
+        </div>
+        </div>
```

Comment:
I would advise against overusing div elements but rather use semantic tags. These will make your code more legible and optimised for SEO. read more about semantic tags here [http://diveintohtml5.it/semantics.html](url)